docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+`ImageNetPipeline` is the class used for inspecting column values after the model has been trained. It has a `string` (`ImagePath`) for the image path. The `Label` is used to create and train the model, and it's also used to evaluate the model. The `PredictedLabelValue` is used during prediction and evaluation. For evaluation, an input with training data, the predicted values, and the model are used.
+
+Add a new class to your project:
+
+1. In **Solution Explorer**, right-click the project, and then select **Add** > **New Item**.
+
+1. In the **Add New Item** dialog box, select **Class** and change the **Name** field to *ImageNetPrediction.cs*. Then, select the **Add** button.
+
+    The *ImageNetPrediction.cs* file opens in the code editor. Remove both the `System.Collections.Generic` and the `System.Text` `using` statements at the top of *ImageNetPrediction.cs*:
+
+Remove the existing class definition and add the following code, which has the `ImageNetPrediction` class, to the *ImageNetPrediction.cs* file:
+
+[!code-csharp[DeclareGlobalVariables](../../../samples/machine-learning/tutorials/TransferLearningTF/ImageNetPrediction.cs#DeclareTypes)]
+
+`ImageNetPrediction` is the image prediction class and has the following fields:
+
+* `Score` contains the confidence percentage for a given image classification.
+* `PredictedLabelValue` contains a value for the predicted image classification label.
+
+`IssuePrediction` is the class used for prediction after the model has been trained. It has a `string` (`ImagePath`) for the image path. The `Label` is used to create and train the model, and it's also used with a second dataset to evaluate the model. The `PredictedLabelValue` is used during prediction and evaluation. For evaluation, an input with training data, the predicted values, and the model are used.
+
+When building a model with ML.NET, you start by creating an <xref:Microsoft.ML.MLContext>. `MLContext` is comparable conceptually to using `DbContext` in Entity Framework. The environment provides a context for your ML job that can be used for exception tracking and logging.
+
+### Initialize variables in Main
+
+Initialize the `_mlContext` global variable  with a new instance of `MLContext`.  Replace the `Console.WriteLine("Hello World!")` line with the following code in the `Main` method:
+
+[!code-csharp[CreateMLContext](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#CreateMLContext)]
+
+## Load the data
+
+ As the input and output of [`Transforms`](../basic-concepts-model-training-in-mldotnet.md#transformer), a `DataView` is the fundamental data pipeline type, comparable to `IEnumerable` for `LINQ`.
+
+In ML.NET, data is similar to a `SQL view`. It is lazily evaluated, schematized, and heterogenous. The object is the first part of the pipeline, and loads the data.  The `DataView` is used to create and train the model.
```

Comment:
"lazily evaluated, schematized, and heterogenous" What does this mean to an average .NET developer? 
DataView is not used to create and train the model. 

Please follow the recommendation tutorial.